addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -752,6 +752,41 @@ QUnit.module("Views", (hooks) => {
         assert.containsOnce(target, ".modal .o_form_view .o_field_widget[name=p]");
     });
 
+    QUnit.test("fieldDependencies are readonly by default", async function (assert) {
+        class MyField extends CharField {}
+        fieldRegistry.add("my_widget", {
+            component: MyField,
+            fieldDependencies: [
+                { name: "int_field", type: "integer" },
+                { name: "bar", type: "boolean" },
+            ],
+        });
+        await makeView({
+            type: "form",
+            resModel: "partner",
+            serverData,
+            arch: `
+                <form>
```

Comment:
add a field no readonly and add it in fieldDependencies

add in message about the write, we don t want to send it in the write